sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSub/src/Trigger/WebPubSubTriggerAttribute.cs

```diff
@@ -86,5 +86,10 @@ public WebPubSubTriggerAttribute(WebPubSubEventType eventType, string eventName)
         /// Allowed service upstream ConnectionString for Signature checks.
         /// </summary>
         public string[] Connections { get; }
+
+        /// <summary>
+        /// Specifies which client protocol can trigger the Web PubSub trigger functions. By default, it accepts all client protocols.
+        /// </summary>
+        public WebPubSubTriggerClientProtocol ClientProtocol { get; set; } = WebPubSubTriggerClientProtocol.All;
```

Comment:
SupportedClientProtocols and use Flag?

Reply:
AcceptedClientProtocols? See `WebPubSubTriggerDispatcher.TryResolveListner` for how to resolve the listeners, basically it's based on string match, so flag doesn't help.